<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="resultsTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>841</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>File</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="addPCAP">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add PCAP File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removePCAP">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove PCAP</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="filterTable">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>521</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <column>
     <property name="text">
      <string>Nickname</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Filter Text</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="addFilter">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteFilter">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchNow">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Now</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="liveSearch">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>36</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Live Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
